### ISSUE TITLE
Review of the lab9 solution - done by Francesca Zafonte

### ISSUE THREAD

**Issue by Zafonte** [ivanmag22/computational-intelligence#5]
Hi Ivan, I reviewed your code and I'm glad to share my feedback.

**_README:_**
The README file is well written and very explanatory of the problem.
I find very useful to read the reasoning and steps you took to solve the problem.

**_CODE:_**
- The code is well formatted and easily readable. The names of variables, functions and comments are descriptive.

- Recombination mechanisms: I really appreciated the implementation of different recombination functions. 

- Unfortunately I have not implemented my own solution to the problem so I have no suggestions to give you for better results. 
   The only thing I would suggest is to add some graphs in order to have a vision of the results obtained.

To summarize, I think that you did a great job and I really appreciated it!

**Comment by ivanmag22** [comment]
Hi Francesca!
You are right! Some plots and graphs miss: they could explain better the results through the generations and the trials.
Thank you for the feedback!
